src/App.vue

```diff
@@ -2,32 +2,47 @@
   <div id="app">
     <div
       ref="wrapper"
-      class="min-h-screen wrapper"
-    >
+      class="min-h-screen wrapper">
+      <div
+        v-if="alert"
+        style="text-align: center;line-height: 5rem;">
+        Unsupported browser.
+      </div>
       <router-view />
     </div>
   </div>
 </template>
 
 <script>
 import { mapActions, mapGetters } from 'vuex';
+import { detect } from 'detect-browser';
 import aeternity from './utils/aeternity';
 import { wallet } from './utils/walletSearch';
 import Backend from './utils/backend';
 import { EventBus } from './utils/eventBus';
-import Util from './utils/util';
+import Util, { IS_MOBILE_DEVICE } from './utils/util';
 
 export default {
   name: 'App',
   data() {
     return {
       page: 1,
+      browser: detect(),
+      alert: false,
     };
   },
   computed: {
     ...mapGetters(['settings', 'tipSortBy']),
+    supportedBrowsers() {
+      return ['chrome', 'firefox'];
+    },
   },
   async created() {
+    if (!IS_MOBILE_DEVICE && this.browser) {
```

Comment:
I would have put all that in 'computed' it is just a 'return' value
like:
```
isSupportedBrowser(){
const browser = detect();
return IS_MOBILE_DEVICE || (browser && supportedBrowsers.includes(browser.name));
}
```
also `const supportedBrowsers = [...] ` should be put before(outside) `export default {` (tough best would be to have that imported from some const file, but not sure we use that elsewhere)